tests/e2e/config/test_path_filtering/slither.config.json

```diff
@@ -2,6 +2,6 @@
     "detectors_to_run": "all",
     "exclude_informational": true,
     "exclude_low": true,
-    "fail_pedantic": false,
+    "fail_on": "none",
```

Comment:
I think this is different than the current default and may mess up CI usage 
cc @elopez

Reply:
this is config for a test though, right? it's not a default

Reply:
```python

    if key.startswith("fail_") and getattr(args, "fail_on") == defaults_flag_in_config["fail_on"]:
        if key == "fail_pedantic":
            pedantic_setting = elem
            fail_on = FailOnLevel.PEDANTIC if pedantic_setting else FailOnLevel.NONE
            setattr(args, "fail_on", fail_on)
            logger.info(f"Migrating fail_pedantic: {pedantic_setting} as fail_on: {fail_on.value}")
```

The original migration was like this so I think `fail_pedantic: False` was migrated to `fail_on: None`